### ISSUE TITLE
Render null links as plain text [enhancement]

### ISSUE THREAD

**Issue by hubertnnn** [andrewicarlson/nova-link-field#4]
I have a use case where link is optional.
Could you make it so if href is null then it will show plain text (or a span) instead of link?

**Comment by spekulatius** [comment]
Yeah, that's my case too. I got Draft items which I don't want link and published items which I would like to link. What are your thoughts @andrewicarlson, is there way do it ?
